### ISSUE TITLE
A11y_SQLAzureDataStudio_SchemaCompare_SelectSource-Target_ScreenReader: The ordering of the radio buttons present in the target section is incorrect.

### ISSUE THREAD

**Issue by v-atkulk** [microsoft/azuredatastudio#9300]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”
#A11yMAS;#A11yTCS;#SQL Azure Data Studio;#DesktopApp;#Win32;#MAS1.3.1;#JAWS;#NVDA; #Benchmark;#A11y_SQLAzureDataStudioBenchmark;#A11ySev2;

**Environment Details:**
Version: 1.16.0-insider
Commit: 506c6a5e5f8423d3249670f55b1466b1980a5f03
Date: 2020-02-17T05:43:18.399Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
Screen reader: NVDA(2019.3.1), JAWS
MAS reference: MAS1.3.1

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Schema Compare extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Schema compare from Context menu.
8. Navigate till Select Source or Select Target & hit enter.
9. Check if proper grouping is assigned for all the radio buttons present on the screen.

**Actual:**
Grouping of the radio buttons is not appropriate, although 2 radio buttons are assigned for "Source schema compare" selection and other 2 buttons for "target schema compare" selection all the buttons are combined into a single group of 4 radio buttons.
Screen reader reads all the radio buttons as 
1 of 4
2 of 4
3 of 4
4 of 4

Issue observed in JAWS and NVDA screen readers.
**Expected:**
Screen reader should announce the radio buttons as:
for Source Schema Compare:
1. Data-tier application File (.dacpac) radio button 1 of 2.
2. Database radio button 2 of 2.
for Target Schema compare:
1. Data-tier application File (.dacpac) radio button 1 of 2.
2. Database radio button 2 of 2.

**User Impact:**
Screen reader users might get confused as all the four radio buttons are combined into a single group and announced as 1 of 4, 2 of 4, 3 of 4, 4 of 4.
Where as 2 radio buttons are for "source schema compare" selection and other 2 for "target schema compare" selection.

**Attachment link reference:**
[9300_A11y_SQLAzureDataStudio_SchemaCompare_SelectSource-Target_ScreenReader.zip](https://github.com/microsoft/azuredatastudio/files/4291424/9300_A11y_SQLAzureDataStudio_SchemaCompare_SelectSource-Target_ScreenReader.zip)

**Comment by alanrenmsft** [comment]
good catch. radio button grouping not set properly

**Comment by kisantia** [comment]
Voiceover announces this correctly. Need to check if something else needs to be set for nvda and narrator to get the grouping correct.

**Comment by Harika1702** [comment]
<img width="960" alt="9300_A11y_Issue resolved" src="https://user-images.githubusercontent.com/100270074/217800026-732a40f1-469e-4ec1-b3fb-8afff8999e3b.png">
Verified this issue in the below environment.
App name: Azure Data studio
Version: 1.41.1 (user setup)
Commit: 33dce95b152227c4f10f6251ab30690fe9895c0c
Date: 2023-01-28T00:46:39.274Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22623
GitHub Tags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
